### DISCUSSION TITLE
Orca slicing slower gcode than Prusa ?

### DISCUSSION THREAD

**Discussion by sergio-tm** [OrcaSlicer/OrcaSlicer#3093] [Q&A]
Hello Everybody, 

I am trying to create Orca profile for my new Anycubic Kobra 2 Neo printer. As it comes with profiles for Cura and Prusa slicers only, I used Prusa settings to create Orca's profile based on Anycubic Kobra 2 presets in Orca. I noticed, that for the same speed and acceleration settings, Orca ( v.1.8.1) shows reasonable longer printing time comparing to Prusa (v.2.7.0) for Inner and Outer walls. Eg. for 3DBenchy, it is 

| Parameter in Orca | Orca time | Parameter in Prusa | Prusa time |
| :---: | :---: | :---: | :---: | 
| Inner Wall | 21m2s | Perimeter | 14m |
| Outer Wall | 13m4s | External Perimeter | 12 m| 

The total print time difference is 56m25s for Orca and 46m for Prusa for 3DBenchy model. For bigger models, the time difference will increase. 

I am wondering if anybody has any ideas if this is expected behaviour or I still have a chance to adjust some settings for Orca to make it more similar with Prusa from the printing time perspective ?

Regards, 
Sergey.

**Comment by Snak3Eyes88** [comment]
Had the same realization today. In Prusa, the pressure takes 6:21 hours and in Orca 9:14 hours
I've compared all the speeds and can't figure out what's wrong.

**Comment by 4eIIIuP** [comment]
Are the "Jerk" installed identically? Additional settings such as slowing down on overhangs, limiting the maximum volumetric speed too?

**Comment by 4eIIIuP** [comment]
![image](https://github.com/SoftFever/OrcaSlicer/assets/158848753/154d0c20-3ae2-437d-ba5a-af61cdb8e643)
![image](https://github.com/SoftFever/OrcaSlicer/assets/158848753/688fa735-8412-4785-bfbd-e6a92fcd67d6)
In general, what I noticed is that in orca slicer, the solid filling is prepared in a slightly different way.
![image](https://github.com/SoftFever/OrcaSlicer/assets/158848753/d69739d2-f41b-4f6a-8445-c928186e4414)
![image](https://github.com/SoftFever/OrcaSlicer/assets/158848753/203abb6a-aaf4-4a3f-b0c5-2902670a4052)
 Further, you can also notice in different fillings, such as 3d honeycombs, that the filling is completely circled around the entire perimeter. at the same time, there is no such thing in orca slicer.
